lib/auth/auth.go

```diff
@@ -2776,6 +2776,8 @@ func (a *Server) CreateWebSession(ctx context.Context, user string) (types.WebSe
 }
 
 // GenerateToken generates multi-purpose authentication token.
+// Deprecated: REMOVE IN 14.0.0 - Clients should now use CreateToken or
+// UpdateToken.
```

Comment:
```suggestion
// Deprecated: Use CreateToken or UpdateToken.
// DELETE IN 14.0.0, replaced by methods above (strideynet).
```

"DELETE IN" vs "REMOVE IN", so greps are consistent.